dart/generate-website-screenshot/lib/main.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+Future<void> start(final req, final res) async {
+  Map<String, dynamic> data = jsonDecode(req.payload);
```

Comment:
Would you please add error handling here too? If an invalid JSON string is passed for the payload, execution just hangs and then times out:

```shell
$ curl -H "X-Internal-Challenge: secret-key" -H "Content-Type: application/json" -X POST http://localhost:3000/ -d '{"payload": "}"}'
```

and the container logs shows:

```
FormatException: Unexpected character (at character 1)
}
^
 #0      _ChunkedJsonParser.fail (dart:convert-patch/convert_patch.dart:1405)
#1      _ChunkedJsonParser.parse (dart:convert-patch/convert_patch.dart:929)
#2      _parseJson (dart:convert-patch/convert_patch.dart:40)
#3      JsonDecoder.convert (dart:convert/json.dart:612)
#4      JsonCodec.decode (dart:convert/json.dart:216)
#5      jsonDecode (dart:convert/json.dart:155)
#6      start (package:generate_website_screenshot/main.dart:5)
#7      main.<anonymous closure>.<anonymous closure> (file:///usr/local/src/server.dart:37)
#8      main.<anonymous closure>.<anonymous closure> (file:///usr/local/src/server.dart:36)
#9      _rootRun (dart:async/zone.dart:1426)
#10     _CustomZone.run (dart:async/zone.dart:1328)
#11     _runZoned (dart:async/zone.dart:1861)
#12     runZonedGuarded (dart:async/zone.dart:1849)
#13     main.<anonymous closure> (file:///usr/local/src/server.dart:35)
<asynchronous suspension>
#14     handleRequest (package:shelf/shelf_io.dart:115)
<asynchronous suspension>
```